app/models/event_schedule.rb

```diff
@@ -37,6 +39,16 @@ def replacement?
 
   private
 
+  def start_after_end_hour
+    return unless event && start_time && event.program && event.program.conference && event.program.conference.end_hour
+    errors.add(:start_time, "can't be after the conference end hour (#{event.program.conference.end_hour})") if start_time.strftime('%H').to_i > event.program.conference.end_hour
```

Comment:
and it should be `>=` as if the conference ends at 19:00, yo don't want any event scheduled at 19:00